src/adaptive.jl

```diff
@@ -0,0 +1,93 @@
+mutable struct Adaptor
+    accepted::Integer
+    total   ::Integer
+    tune    ::Integer   # tuning interval
+    target  ::Float64   # target acceptance rate
+    bound   ::Float64   # bound on logσ of Gaussian kernel
+    δmax    ::Float64   # maximum adaptation step
+end
+
+Adaptor(; tune=25, target=0.44, bound=10., δmax=0.2) =
+    Adaptor(0, 0, tune, target, bound, δmax)
```

Comment:
```suggestion
function Adaptor(; tune=25, target=0.44, bound=10., δmax=0.2)
    return Adaptor(0, 0, tune, target, bound, δmax)
end
```

Multiline functions are more commonly written in `function ... end` syntax.